core/genesis_test.go

```diff
@@ -270,8 +271,7 @@ func TestPrecompileActivationAfterHeaderBlock(t *testing.T) {
 	contractDeployerConfig := deployerallowlist.NewContractDeployerAllowListConfig(big.NewInt(51), nil, nil)
 	activatedGenesis.Config.UpgradeConfig.PrecompileUpgrades = []params.PrecompileUpgrade{
 		{
-			// Enable ContractDeployerAllowList at timestamp 50
-			ContractDeployerAllowListConfig: contractDeployerConfig,
+			StatefulPrecompileConfig: contractDeployerConfig,
```

Comment:
not sure I like the name "Stateful" that much, what value does that prefix add?

Reply:
changed it to Config